server/reflector/pipelines/main_multitrack_pipeline.py

```diff
@@ -0,0 +1,749 @@
+                self.logger.warning(f"Track {idx} processing failed")
+
+        transcript.data_path.mkdir(parents=True, exist_ok=True)
+
+        # Mixdown tracks directly from URLs
+        mp3_writer = AudioFileWriterProcessor(
+            path=str(transcript.audio_mp3_filename),
+            on_duration=self.on_duration,
+        )
+        await self.mixdown_tracks(padded_track_urls, mp3_writer, offsets_seconds=None)
+        await mp3_writer.flush()
+
+        if not transcript.audio_mp3_filename.exists():
+            raise Exception(
+                "Mixdown failed - no MP3 file generated. Cannot proceed without playable audio."
+            )
+
+        mp3_data = transcript.audio_mp3_filename.read_bytes()
+        storage_path = f"{transcript.id}/audio.mp3"
+        await storage.put_file(storage_path, mp3_data)
+        mp3_url = await storage.get_file_url(storage_path)
+
+        await transcripts_controller.update(transcript, {"audio_location": "storage"})
+
+        self.logger.info(
+            f"Uploaded mixed audio to storage",
+            storage_path=storage_path,
+            size=len(mp3_data),
+            url=mp3_url,
+        )
+
+        try:
+            self.logger.info("Generating waveform from mixed audio")
+            waveform_processor = AudioWaveformProcessor(
+                audio_path=transcript.audio_mp3_filename,
+                waveform_path=transcript.audio_waveform_filename,
+                on_waveform=self.on_waveform,
+            )
+            waveform_processor.set_pipeline(self.empty_pipeline)
+            await waveform_processor.flush()
+            self.logger.info("Waveform generated successfully")
+        except Exception as e:
+            self.logger.error("Waveform generation failed", error=str(e), exc_info=True)
+
+        # Transcribe each padded track directly from URLs
+        speaker_transcripts: list[TranscriptType] = []
+        for idx, padded_url in enumerate(padded_track_urls):
+            if not padded_url:
+                continue
+
+            try:
+                t = await self.transcribe_file(padded_url, transcript.source_language)
+            except Exception as e:
+                self.logger.error(
+                    "Transcription via default backend failed, trying local whisper",
+                    track_idx=idx,
+                    url=padded_url,
+                    error=str(e),
+                )
+                try:
+                    t = await transcribe_file_with_processor(
+                        padded_url, transcript.source_language, processor_name="whisper"
+                    )
+                except Exception as e2:
+                    self.logger.error(
+                        "Skipping track - transcription failed after fallback",
+                        track_idx=idx,
+                        url=padded_url,
+                        error=str(e2),
+                    )
+                    continue
```

Comment:
No, no fallback on local transcription ? We don't do that anywhere else, and never discussed. Please don't :D
And if it fail, it should just raise, to be retried later.